go/vt/sqlparser/sql_test.go

```diff
@@ -280,6 +280,264 @@ func TestShowIndex(t *testing.T) {
 	}
 }
 
+func TestCreateForeignKey(t *testing.T) {
+	tests := []testForeignKeyStruct{
```

Comment:
Great tests. Would add one more, for the base syntax (no constraint keyword, unnamed constraint) without any ON UPDATE ON DELETE clauses.